core-tests/src/test/java/org/mule/runtime/core/el/DefaultExpressionManagerTestCase.java

```diff
@@ -208,68 +142,27 @@ public void parseCompatibility() throws MuleException {
     assertThat(expressionManager.parse("dw:'this is ' ++ payload", testEvent(), mock(FlowConstruct.class)),
                is(String.format("this is %s", TEST_PAYLOAD)));
     expectedException.expect(RuntimeException.class);
-    expressionManager.parse(weavify(expression), testEvent(), mock(FlowConstruct.class));
+    expressionManager.parse("dw:" + expression, testEvent(), mock(FlowConstruct.class));
```

Comment:
Because this is the only scenario in which the manager has to do a prefix check so it didn't make sense to have a method for it.